week_1/main.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>week_1 Assignment</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+<body>
+    <h1 style="padding-left:40px;">
+        My Guest Book
+    </h1>
+    <p>
+        <div class="first line" style="padding-left:40px;">
+            Please sign my guest book. Thanks!
+        </div>
+    </p>
+    <h3 style="padding-left:40px;">
```

Comment:
You have used `padding-left:40px` multiple times. You can instead turn it into a class to avoid repeating the code.

Reply:
Will implement this in future code. Thank you.